pkg/cmd/login/html/html.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package html
+
+import (
+	"html/template"
+	"net/http"
+
+	"github.com/okteto/okteto/pkg/okteto"
+)
+
+type templateData map[string]interface{}
+
+// ExecuteSuccess renders the login success page at the browser
+func ExecuteSuccess(w http.ResponseWriter) error {
+	data := successData()
+	successTemplate := template.Must(template.New("success-response").Parse(commonTemplate))
```

Comment:
We should move the `template.Must` to the module initialization, so we crash early if the template is malformed. Same thing in `ExecuteError`

Reply:
Adding a test should be another easy win, since the content is static

Reply:
I think it should stay in a function, otherwise, it'll be loaded when importing the package, even if we don't use the function.

Reply:
In this case I think it is not a big deal to fail with a panic as it is not a server that would crash and generate a restart like our API. This is just a command execution, the only thing to bear in mind the user experience. I think users should not see a `panic`.

Having said that, I don't see the need of doing a `Must`, we can simply use `successTemplate, err := template.New("success-response").Parse(commonTemplate)` and return an error if it is not nil (as `ExecuteSuccess` already may return an error. 

As the `Must` function documentation says:

```
// Must is a helper that wraps a call to a function returning (*Template, error)
// and panics if the error is non-nil. It is intended for use in variable
// initializations such as
//	var t = template.Must(template.New("name").Parse("text"))
```